lib/OpenQA/Markdown.pm

```diff
@@ -25,30 +19,46 @@ sub is_light_color {
     return $sum > 380;
 }
 
-sub _bugref_to_html ($bugref) {
-    my $bugurl = bugurl($bugref);
-    return "<a href=\"$bugurl\">$bugref</a>";
+sub bugref_to_html ($bugref, $fancy = 0) {
+	my $app = OpenQA::App->singleton;
+    my $bugs = $app->schema->resultset('Bugs');
+	my $bug = $bugs->get_bug($bugref);
+    my $bugurl = $app->bugurl_for($bugref);
+    my $bugtitle = $app->bugtitle_for($bugref, $bug);
+    my $bugicon = $app->bugicon_for($bugref, $bug);
+	if ($fancy) {
+		return qq{<span title="$bugtitle" class="openqa-bugref"><a href="$bugurl"><i class="test-label $bugicon"></i>&nbsp;$bugref</a></span>};
+	}
```

Comment:
Maybe using a suffix if would make sense. Or combine it with the next statement using the ternary operator.